themes/Clearlooks/lxqt-panel.qss

```diff
@@ -1,576 +1,815 @@
-    border: 0px solid #1a1a1a;
+LXQtPanel, LXQtPanel #BackgroundWidget {
+    color: #1a1a1a;
+    background: #edeceb;
+}
+
+/* Panel Borders by position */
+LXQtPanel[position="Bottom"] #BackgroundWidget { border-top: 1px solid #ffffff; }
+LXQtPanel[position="Left"] #BackgroundWidget { border-right: 1px solid #ffffff; }
+LXQtPanel[position="Right"] #BackgroundWidget { border-left: 1px solid #ffffff; }
+
+/* Catch-all settings for panel plugins and their subwidgets 
+    not otherwise styled in this theme: 
+    #ColorPicker
+    #Custom and #CustomButton
+    #DirectoryMenu
+    #Dom
+    #LXQtMount
+    #NetworkMonitor
+    #Sensors
+    #ShowDesktop
+    #Spacer
+*/ 
+Plugin, Plugin > QToolButton, Plugin > QWidget > QToolButton,
+Plugin > QWidget > QToolButton > QWidget > QToolButton {
+    color: #1a1a1a;
+    background: #edeceb;
+    padding: 0px;
+    margin: 0px;
+    /* prevents unexpected borders/boxes on hover */
+    border: 0px solid transparent;
 }
 
-Plugin > QWidget,
-Plugin > QWidget > QWidget {
-    border: 0px;
-    color: rgba(0, 0, 0, 100%); /* Color of text on same */
+QToolTip {
+    color: #000000;
+    background: #e7f3fd;
+    border: 1px solid #e7f3fd;
+    padding: 1px;
 }
 
-Plugin > QToolButton,
-Plugin > QWidget > QToolButton,
-Plugin > QWidget > QToolButton > QWidget > QToolButton,
-
-LXQtPanelPlugin > QToolButton {
-    background: transparent;
-    margin: 0px;
-    padding: 2px;
+#TaskBar,
+#DesktopSwitch,
+#WorldClock,
+#WorldClockContent {
     color: #1a1a1a;
+    background: transparent;
 }
 
-
-QToolTip {
-    border: 1px solid palette(text);
-    background: palette(window);
-    color: palette(text);
-    padding: 1px;
+#QuickLaunch,
+#QuickLaunchPlaceHolder {
     margin: 0px;
 }
 
-/* Menus */
+#StatusNotifierPlugin,
+#StatusNotifierPlugin > QToolButton,
+#StatusNotifierPlugin StatusNotifierButton {
+    padding: 2px;
+    margin-right: 1px;
+}
 
-QMenu {
-    background: #ffffff;
+#MainMenu,
+#FancyMenu {
     color: #1a1a1a;
-    border: 1px solid #1a1a1a;
-    padding: 1px;
+    background: #edeceb;
+    qproperty-icon: url(assets/menus/mainmenu.svg);
+    border: 1px solid transparent;
+    border-radius: 3px;
+    padding-left: 4px;
+    margin: 1px;
 }
 
-QMenu QToolButton {
-    background-color: #80a0d0;
-    color: white;
-    /* padding and border guarantee that it won't be drawn by widget style */
-    padding: 3px;
-    border: none;
-    font: bold;
+#MainMenu:hover,
+#FancyMenu:hover {
+    color: #1a1a1a;
+    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #ffffff, stop:0.49 #fbfbfa, stop:0.50 #f5f5f4, stop:1 #e8e7e6);
+    border: 1px solid #918e8c;
 }
 
-QMenu::icon {
-    padding-right: 3px;
+#MainMenu:pressed,
+#FancyMenu:pressed {
+    color: #1a1a1a;
+    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #ccc9c7, stop:0.49 #cfcdcb, stop:0.50 #cbc8c6, stop:1 #c0bdba);
+    border: 1px solid #888582;
 }
 
-QMenu::icon:selected {
-    padding-right: 2px;
```

Comment:
This makes the selected item icon in the panel menu (panel settings, close window ecc) move to the left which doesn't look fine. It's present in the original theme too, also for the mainmenu. It's probably intended but IMO color change is enough.